webapp/src/components/end_call_modal/component.tsx

```diff
@@ -0,0 +1,205 @@
+import React, {CSSProperties} from 'react';
```

Comment:
I think we could do this a bit simpler by using the generic modal we just added. Probably only a dozen lines of code or so. If you agree, not sure if you want to do that now or leave it for later? Up to you.

Reply:
Agreed. I'll have a look and create a separate ticket if it becomes a blocker.

Reply:
Had a good look and almost implemented it so it's totally doable but then there's a bit of time to spend on styling it properly, centering content and such so I am looking to defer this work for later as we need to convert a few of these anyway. Created [MM-44708](https://mattermost.atlassian.net/browse/MM-44708) to keep track of this work.